application/src/main/java/org/togetherjava/tjbot/commands/help/LeaversPurgeDatabaseRoutine.java

```diff
@@ -0,0 +1,43 @@
+package org.togetherjava.tjbot.commands.help;
+
+import net.dv8tion.jda.api.JDA;
+import org.jetbrains.annotations.NotNull;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.togetherjava.tjbot.commands.Routine;
+import org.togetherjava.tjbot.db.Database;
+import org.togetherjava.tjbot.db.generated.tables.HelpThreads;
+import java.time.Instant;
+import java.time.Period;
+import java.util.concurrent.TimeUnit;
+
+/**
+ * Purge Routine to get rid of old thread creations in the database.
+ */
+public class LeaversPurgeDatabaseRoutine implements Routine {
```

Comment:
This is still called `Leavers...` but now the database table is for help threads in general. Id rename it to `HelpThreadMetadataPurger` or similar